### PR TITLE
HEEDLS-235 Show times on section cards back end

### PR DESCRIPTION
**Pull request by kwunkiuma**
Add extra fields to the query and class.

Amend the tests to use the new constructor.

Ran and passed all unit tests.

### CODE REVIEW

File: DigitalLearningSolutions.Data/Models/CourseContent/CourseSection.cs [hunk 1]
```diff
@@ -6,13 +6,23 @@ public class CourseSection
         public int Id { get; }
         public bool HasLearning { get; }
         public double PercentComplete { get; }
+        public int TimeMins { get; }
+        public int AverageSectionTime { get; }
 
-        public CourseSection(string sectionName, int id, bool hasLearning, double percentComplete)
+        public CourseSection(string sectionName,
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data/Models/CourseContent/CourseSection.cs:12]: section name should be on a new line

File: DigitalLearningSolutions.Data/Services/CourseContentService.cs [hunk 2]
```diff
@@ -51,7 +51,9 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
                                  OR dbo.CheckCustomisationSectionHasLearning(Customisations.CustomisationID, Sections.SectionID) = 0
                             THEN 0
                             ELSE CAST(SUM(aspProgress.TutStat) * 100 AS FLOAT) / (COUNT(Tutorials.TutorialID) * 2)
-                         END) AS PercentComplete
+                         END) AS PercentComplete,
+                         SUM(aspProgress.TutTime) as TimeMins,
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data/Services/CourseContentService.cs:55]: I assume this returns something sensible if there's no progress for the candidate? 🙂

**Review by FridaTveit — COMMENTED** [review]
A couple of comments, otherwise looks good 🙂

File: DigitalLearningSolutions.Data/Services/CourseContentService.cs (hunk 2 above)
Comment (kwunkiuma) [reply DigitalLearningSolutions.Data/Services/CourseContentService.cs:55]: It currently returns null, I'll change it so that it defaults to 0.

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**